db-move

```diff
@@ -103,6 +105,18 @@ for pkgbase in "${args[@]:2}"; do
 					fi
 					add_pkgs+=("${FTP_BASE}/${PKGPOOL}/${pkgfile}")
 					remove_pkgs+=("${pkgname}")
+
+					debug_pkgpath=$(getpkgfile "${ftppath_debug_from}/${tarch}/${pkgname}-debug-${pkgver}-${pkgarch}"${PKGEXTS})
```

Comment:
We need to do this only once, and do it using pkgbase not pkgname -- current versions of makepkg create one debug package per PKGBUILD.

This is relevant in case the pkgbase differs from all the pkgnames, e.g.
```
pkgbase=pyqt5
pkgname=('python-pyqt5')
```

(formerly also python2-pyqt5)

Reply:
Moved this out of the `pkgnames` loop used `pkgbase` instead.